src/Core/NotificationsManager.php

```diff
@@ -461,19 +424,21 @@ public function systemNotifs($seen = 0, $start = 0, $limit = 80)
 		$sql_seen = "";
 
 		if ($seen === 0) {
-			$sql_seen = " AND NOT `seen` ";
+			$filter = ['`uid` = ? AND NOT `seen`', local_user()];
+		} else {
+			$filter = ['uid' => local_user()];
```

Comment:
Please rewrite this block like this:
```php
$filter = ['uid' => local_user()];
if ($seen === 0) {
    $filter['seen'] = false;
}
```

Reply:
So `false` will be interpreted as  DB - `NOT NULL`. That was the missing link :-) thanks

Reply:
No, it will be interpreted as ``NOT `field` ``